Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj

```diff
@@ -315,23 +315,25 @@
     <AndroidResource Include="Resources\drawable\mic.png" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json">
-      <Version>10.0.3</Version>
-    </PackageReference>
-    <PackageReference Include="Xam.Plugin.DeviceInfo">
-      <Version>3.0.2</Version>
-    </PackageReference>
-    <PackageReference Include="Xamarin.Build.Download">
-      <Version>0.4.9</Version>
-    </PackageReference>
-    <PackageReference Include="Xamarin.Firebase.AppIndexing">
+  <PackageReference Include="Newtonsoft.Json">
+    <Version>10.0.3</Version>
+  </PackageReference>
+  <PackageReference Include="Xam.Plugin.DeviceInfo">
+    <Version>3.0.2</Version>
+  </PackageReference>
+  <PackageReference Include="Xamarin.Build.Download">
+    <Version>0.4.11</Version>
+  </PackageReference>
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFrameworkVersion)' == 'v8.1' ">
+      <PackageReference Include="Xamarin.Firebase.AppIndexing">
       <Version>60.1142.1</Version>
     </PackageReference>
     <PackageReference Include="Xamarin.Firebase.Common">
       <Version>60.1142.1</Version>
     </PackageReference>
-    <PackageReference Include="Xamarin.GooglePlayServices.AppIndexing">
-      <Version>60.1142.1</Version>
+    <PackageReference Include="Xamarin.GooglePlayServices.AppIndexing">	
+      <Version>60.1142.1</Version>	
```

Comment:
Can we bump the other GPS packages to newest too? I suspect there may be dependency conflict problems otherwise.

Reply:
That's what I did, right?

Note that there are two branches now: when droid8.1 is targeted I use the old packages since they are not compatible with droid9 and vice versa.

The specific package you mention here isn't in the droid9 branch because it states to be just a package for backwards compatibility and you should use the FireBase.AppIndexing one, that one is included.

Am I still missing something then?

Reply:
I think app links was even broken  so we might as well update and test it.